go-controller/go.mod

```diff
@@ -58,6 +58,7 @@ require (
 	kubevirt.io/api v1.0.0-alpha.0
 	sigs.k8s.io/controller-runtime v0.15.1
 	sigs.k8s.io/network-policy-api v0.1.1
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3
```

Comment:
Out of curiosity, what does this do?

Reply:
it is used by `applyconfiguration`, e.g. here https://github.com/ovn-org/ovn-kubernetes/pull/3750/commits/5c707cdd6d6d6d992c5f93e8f1f9f498622ae702#diff-a56cb620ea0baa29f6de1f61806fecded83591d3c2339ebac00e7da47411ca54R24

Reply:
I can't see this anymore after the latest push

Reply:
that's because it points to the collapsed file :( check [go-controller/pkg/crd/adminpolicybasedroute/v1/apis/applyconfiguration/internal/internal.go](https://github.com/ovn-org/ovn-kubernetes/commit/5c707cdd6d6d6d992c5f93e8f1f9f498622ae702#diff-a56cb620ea0baa29f6de1f61806fecded83591d3c2339ebac00e7da47411ca54)